frontend/src/store/modules/visitors/actions.js

```diff
@@ -30,6 +31,29 @@ export default {
             context.commit(RESET_BUTTON_FETCHING, data.button);
         }
     },
+    [GET_LINE_CHART_DATA]: (context, data) => {
```

Comment:
don't call action as getter, unless it returns anything, in this particular case - it doesn't and shouldn't return anything